courses/fundamentals_of_ada/130_program_structure.rst

```diff
@@ -635,42 +635,31 @@ Visibility Limits
     -- Parent_T is visible here
    end Parent.Child;
 
---------------------------
-Misbehaving (?) Children
---------------------------
+--------------------------------
+Children Can Break Abstraction
+--------------------------------
 
-* Can break a parent's abstraction
+* Could **break** a parent's abstraction
 
-  - Exporting a subprogram that alters package state
-  - Exporting a subprogram that alters ADT object state
+  - Alter a parent package state
+  - Alters an ADT object state
 
-* Nice for testing via fault injection...
+* Useful for testing: fault injections...
```

Comment:
Useful for
   Reset data during error handling
   Testing (fault injections, data queries)